themes/src/main/resources/theme/keycloak-preview/account/resources/app/account-service/account.service.ts

```diff
@@ -102,7 +102,7 @@ class AccountServiceClient {
 
         // add request params
         if (config && config.hasOwnProperty('params')) {
-            const params: {[name: string]: string} = config.params as {};
+            const params: {[name: string]: string} = config.params as {} || {};
```

Comment:
Is this a typo or is it some idiom I've never seen?

Reply:
no it's not a typo, it could be undefined so when it is it will take an empty object